clang/include/clang/AST/Type.h

```diff
@@ -4068,7 +4075,7 @@ class FunctionType : public Type {
 
     /// Any AArch64 SME ACLE type attributes that need to be propagated
     /// on declarations and function pointers.
-    unsigned AArch64SMEAttributes : 6;
+    unsigned AArch64SMEAttributes : 8;
```

Comment:
We could claw back these two bits by reducing `NumExceptionType` from 10 to 8.

Reply:
Note that doing so is perhaps a breaking change for users, it DOES put us right up at 'imp-limits', but it is entirely possible that users are counting on us exceeding the minimum here.  And this is a reduction to a 'reasonable' number.